src/domain/schemas/shared-schemas.js

```diff
@@ -1,3 +1,5 @@
 import joi from 'joi';
 
 export const idOrKeySchema = joi.string().alphanum().min(15).max(30);
+export const boolStringSchema = joi.any().allow('true', 'false');
```

Comment:
I just moved this here, but I will check.

Reply:
Ah, also it should be joi.string() since the name says "boolString". We discovered yesterday that allow('') adds on top of allowed things while "valid" restricts the possibilities to exactly that which is specified.

Reply:
Yup, the documentation regarding `allow` is clear

"Note that this list of allowed values is in addition to any other permitted values. To create an exclusive list of values, see any.valid(value)."

Thanks for pointing it out, changed.

Reply:
> Ah, also it should be joi.string() since the name says "boolString". We discovered yesterday that allow('') adds on top of allowed things while "valid" restricts the possibilities to exactly that which is specified.

I kept it `any` and `valid("true", "false", true, false)`, this way this will work regardless of json parsing having been performed before or not.